main.go

```diff
@@ -30,12 +30,25 @@ func main() {
 		scenariolib.InitLogger(ioutil.Discard, os.Stdout, os.Stdout, os.Stderr)
 	}
 
-	scenariolib.Info.Printf("Queue Length: %v", *queueLength)
-	scenariolib.Info.Printf("Server Port: %v", *port)
+
 
 	source := rand.NewSource(int64(time.Now().Unix()))
 	random := rand.New(source)
 
+	if *queueLength < 1 || *queueLength > 500 {
+		scenariolib.Info.Printf("Queue Length is out of bounds, should be in [1,500].  Set to default value")
```

Comment:
Specify the default value in the log message. Same comment for all messages. If we change the parameter, we need to log to which value it will be set.